crates/containerd-shimkit/src/vendor/containerd_shim/logger.rs

```diff
@@ -87,6 +87,14 @@ impl SimpleWriteVisitor {
     }
 }
 
+pub(super) static KEY_VALUES: Mutex<String> = Mutex::new(String::new());
```

Comment:
Do you plan to upstream this to rust-extensions?

Reply:
We should!